### PR TITLE
Fix: allow max payload of 8b + 1b checksum

### PR DESCRIPTION
**Pull request by abrauchli**

### CODE REVIEW

**Comment by abrauchli** [comment]
Job fails due to dependency on 0.2.0 which is to be released after https://github.com/Sensirion/lin-bus-rs/pull/14

**Comment by rnestler** [comment]
> Job fails due to dependency on 0.2.0 which is to be released after Sensirion/lin-bus-rs#14

I published the release. Note that you could also just point dependencies to a git repository directly. I also invited @abrauchli and @psachs as owners of the lin-bus crate.

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "lin-bus-driver-serial"
-version = "0.1.1"
+version = "0.1.2"
```
Comment (rnestler) [inline Cargo.toml]: Since we bump a public dependency with a semver major bump, we should also bump the version to 0.2 here.

**Review by rnestler — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (abrauchli) [reply Cargo.toml]: yup, makes sense, thanks

**Review by rnestler — APPROVED** [review]

**Pull request merged**